sys/include/net/ng_zep.h

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    net_ng_zep  Zigbee Encapsulation Protocol
+ * @ingroup     net
+ * @brief       Transports IEEE 802.15.4 frames over UDP (can be parsed by
+ *              Wireshark)
+ * @{
+ *
+ * @file
+ * @brief       ZEP definitions
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+
+
+#ifndef NG_ZEP_H_
+#define NG_ZEP_H_
+
+#include <inttypes.h>
+
+#include "byteorder.h"
+#include "kernel_types.h"
+#include "net/ng_ipv6/addr.h"
+#include "net/ng_nettype.h"
+#include "thread.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Maximum possible packet size in byte
+ */
+#define NG_ZEP_MAX_PKT_LENGTH   (116)
+
+/**
+ * @brief   Default stack size to use for the ZEP thread
+ */
+#ifndef NG_ZEP_STACK_SIZE
+#define NG_ZEP_STACK_SIZE       (THREAD_STACKSIZE_DEFAULT)
+#endif
+
+/**
+ * @brief   Default priority for the ZEP thread
+ */
+#ifndef NG_ZEP_PRIO
+#define NG_ZEP_PRIO             (THREAD_PRIORITY_MAIN - 5)
+#endif
+
+/**
+ * @brief   Default message queue size to use for the ZEP thread.
+ */
+#ifndef NG_ZEP_MSG_QUEUE_SIZE
+#define NG_ZEP_MSG_QUEUE_SIZE   (8U)
+#endif
+
+/**
+ * @brief   Default addresses if the CPUID module is not present
+ * @{
+ */
+#define NG_ZEP_DEFAULT_ADDR_SHORT   (0x98b0)
+#define NG_ZEP_DEFAULT_ADDR_LONG    (0x4a8a6377552b4249)
+/**
+ * @}
+ */
+
+/**
+ * @brief   Channel configuration
+ * @{
+ */
+#define NG_ZEP_MIN_CHANNEL      (11U)
+#define NG_ZEP_MAX_CHANNEL      (26U)
+#define NG_ZEP_DEFAULT_CHANNEL  (17U)
```

Comment:
Why do we need to define these values here again? What makes them different from IEEE 802.15.4 defines?